src/Longrunning/OperationsClient.php

```diff
@@ -0,0 +1,478 @@
+<?php
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was generated from the file
+ * https://github.com/google/googleapis/blob/master/google/longrunning/operations.proto
+ * and updates to that file get reflected here through a refresh process.
+ *
+ * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
+ * more frequently than those which have been declared beta or 1.0, including changes which break
+ * backwards compatibility.
+ */
+
+namespace Google\Longrunning;
+
+use Google\GAX\AgentHeaderDescriptor;
+use Google\GAX\ApiCallable;
+use Google\GAX\CallSettings;
+use Google\GAX\GrpcConstants;
+use Google\GAX\GrpcCredentialsHelper;
+use Google\GAX\PageStreamingDescriptor;
+use Google\GAX\ValidationException;
+use google\longrunning\CancelOperationRequest;
+use google\longrunning\DeleteOperationRequest;
+use google\longrunning\GetOperationRequest;
+use google\longrunning\ListOperationsRequest;
+use google\longrunning\OperationsGrpcClient;
+
+/**
+ * Service Description: Manages long-running operations with an API service.
+ *
+ * When an API method normally takes long time to complete, it can be designed
+ * to return [Operation][google.longrunning.Operation] to the client, and the client can use this
+ * interface to receive the real response asynchronously by polling the
+ * operation resource, or pass the operation resource to another API (such as
+ * Google Cloud Pub/Sub API) to receive the response.  Any API service that
+ * returns long-running operations should implement the `Operations` interface
```

Comment:
This is a hard one... Ideally I'd like to have the original comment in the proto reworded given that this will clash with other languages too. Proto comments are written for the proto context, and writers aren't taking into account the target language context. I'm fine if you want to work around it for now, though.